test/supplemental_genbank_tests/genbank_upload_full_test.py

```diff
@@ -167,3 +171,125 @@ def test_simple_shock_upload(self):
         self.assertIsNotNone(result['genome_ref'])
         self.assertTrue(int(
             result['genome_info'][10]['Number of Protein Encoding Genes']) > 0)
+
+    def _run_test_fail(self, params, error_message, mass=True):
+        with pytest.raises(Exception) as got:
+            if mass:
+                self.getImpl().genbanks_to_genomes(self.getContext(), params)
+            else:
+                self.getImpl().genbank_to_genome(self.getContext(), params)
+        assert_exception_correct(got.value, ValueError(error_message))
+
+    def _get_objects_data(self, refs):
+        results = self.getDfuClient().get_objects({'object_refs': refs})
+        return [result["info"] for result in results["data"]]
```

Comment:
This is returning just the object info and throwing away the data. If you wanted to save some time you could use a ws client to get the info with get_object_info3

Reply:
👍

Reply:
Moot now, so :+1: